hpi-schul-cloud/schulcloud-client
# BC-2373 - Mentioning of HPI in School Administration

# Description

QA note:
21.08.2023
Change tested on env: 
https://nbc-bc-2373-mentioning-of-hpi-in-school-administration.cd.dbildungscloud.dev/
Change tested only on NBC due to server settings LDAP.
The "More information" section still needs to be changed.

22.08.2023
Tested again on env: 
https://nbc-bc-2373-mentioning-of-hpi-in-school-administration.cd.dbildungscloud.dev/
No issues found. 
See screen in JIRA ticket: https://ticketsystem.dbildungscloud.de/browse/BC-2373 


## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-2373


## Changes


## Data Security


## Deployment


## New Repos, NPM packages or vendor scripts


## Screenshots of UI changes


## Approval for review


> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.
